### PR TITLE
PUBDEV-5605: Fixing user guide stopping_metric values

### PR DESCRIPTION
**Pull request by ledell**
https://0xdata.atlassian.net/browse/PUBDEV-5605

### CODE REVIEW

**Comment by angela0xdata** [comment]
Oh, there might be a conflict. I just merged a pull request that included a driveby fix to stopping_metric:
https://github.com/h2oai/h2o-3/pull/2460/files#diff-5b36818fe8c291ef3cc9b9ec3c053936

I just added “(mean residual deviance)” beside the first instance of “deviance.”

**Comment by angela0xdata** [comment]
Yep, there it is :)

**Comment by angela0xdata** [comment]
Is "r2" an available stopping_metric now? (See https://github.com/h2oai/h2o-3/pull/2199/files)

File: h2o-docs/src/product/data-science/algo-params/stopping_metric.rst [hunk 1]
```diff
@@ -19,14 +19,14 @@ then the model will stop training after reaching three scoring events in a row i
...
-- ``rmsle``
-- ``auc``
+- ``MSE``
+- ``RMSE``
+- ``MAE``
+- ``RMSLE``
+- ``AUC``
 - ``lift_top_group``
 - ``misclassification``
 - ``mean_per_class_error``
```
Comment (angela0xdata) [inline h2o-docs/src/product/data-science/algo-params/stopping_metric.rst]: r2 ?

**Comment by ledell** [comment]
@angela0xdata You're right!  I just tested it and it does work.  I will made an edit in a follow-up PR.

**Comment by angela0xdata** [comment]
I just added it to Navdeep's branch for https://github.com/h2oai/h2o-3/pull/2480 along with some other doc changes.

**Pull request merged**